Doc/whatsnew/3.14.rst

```diff
@@ -122,6 +122,9 @@ Others
   These had previously raised a :exc:`DeprecationWarning` since Python 3.12.
   (Contributed by Raymond Hettinger in :gh:`101588`.)
 
+* Remove deprecated ``Quoter`` class from :mod:`urllib.parse`.
+  It had previously raised a :exc:`DeprecationWarning` since Python 3.11.
```

Comment:
Please can you move this into a "urllib" or "urllib.parse" section, above the "Others" I added in https://github.com/python/cpython/pull/118799? Let's list the modules alphabetically like in 3.13.